src/System.Text.Json/tests/Utf8JsonWriterTests.cs

```diff
@@ -5785,6 +5845,34 @@ private static string GetNumbersExpectedString(bool prettyPrint, string keyStrin
             return Encoding.UTF8.GetString(ms.ToArray());
         }
 
+        private static string GetExpectedString_RelaxedEscaping(bool prettyPrint, string keyString)
+        {
+            var ms = new MemoryStream();
+            TextWriter streamWriter = new StreamWriter(ms, new UTF8Encoding(false), 1024, true);
+
+            var json = new JsonTextWriter(streamWriter)
```

Comment:
We should discuss whether this is the best way to get expected strings going forward. Seems to work by coincidence in this case (same escaping rules). In other cases we have started to diverge (e.g. casing).